sevntu-checks/sevntu-checks.xml

```diff
@@ -80,7 +80,11 @@
       <property name="forbiddenClassesRegexpProtected" value=""/>
       <property name="forbiddenClassesRegexpPublic" value=""/>
     </module>
-    <module name="com.github.sevntu.checkstyle.checks.coding.OverridableMethodInConstructorCheck">
+    <module name="com.github.sevntu.checkstyle.checks.coding.ForbidCertainMethodsCheck">
+      <property name="methodName" value="exit"/>
+      <property name="argumentCount" value=""/>
+    </module>
+  <module name="com.github.sevntu.checkstyle.checks.coding.OverridableMethodInConstructorCheck">
```

Comment:
This line shouldn't be a change. Looks like the spacing wasn't restored correctly.

Reply:
Fixed.